service/static/js/rest_api.js

```diff
@@ -76,179 +73,4 @@ $(function () {
         ajax.fail(function(res){
             flash_message(res.responseJSON.message)
         });
-    });
```

Comment:
Here's where your error is coming from! You deleted the second half of the rest_api.js file including the update, query, delete, clear form, and search functions.

In doing so, we also lost the bracket and parenthesis `})` which closes the javascript function. This creates a syntax error preventing your file from compiling and running